### [CVE-2008-7264](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2008-7264)
![](https://img.shields.io/static/v1?label=Product&message=n%2Fa&color=blue)
![](https://img.shields.io/static/v1?label=Version&message=n%2Fa%20&color=brightgreen)
![](https://img.shields.io/static/v1?label=Vulnerability&message=n%2Fa&color=brightgreen)

### Description

The ftp_QUIT function in ftpserver.py in pyftpdlib before 0.5.0 allows remote authenticated users to cause a denial of service (file descriptor exhaustion and daemon outage) by sending a QUIT command during a disallowed data-transfer attempt.

### POC

#### Reference
- http://code.google.com/p/pyftpdlib/issues/detail?id=71
- http://code.google.com/p/pyftpdlib/source/browse/trunk/HISTORY

#### Github
No PoCs found on GitHub currently.

